app/policies/case/foi/standard_policy.rb

```diff
@@ -8,7 +8,19 @@ def initialize(user, scope)
 
     def resolve
       if @user.permitted_correspondence_types.include? CorrespondenceType.foi
-        @scope
+        
+        if @user.responder_only?
+          team_restriction_sql = "select case_id from assignments 
```

Comment:
Might be clearer to change the case of the sql here so that it is clearer when parsing the code i.e. `select` => `SELECT` and `join` => `JOIN` etc.